CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+<a name="0.38.0"></a>
+
+## 0.38.0 (2021-03-12)
+
+### Features and enhancements
+
+* [core] Make possible to pass variadic pins to children variadic nodes (#2107)
+* [ide] Add table log feature: collect, view and save tabular data in live session mode (#2098, #2108)
+* [nodes] Add nodes to handle interrupts in [xod/gpio](https://xod.io/libs/xod/gpio) standard library (#2106)
+* [nodes] Add [xod/core/micros](https://xod.io/libs/xod/core/micros) node and some utilities for this type (#2106)
+* [nodes] Add `variadic-pass-*` marker nodes (#2107)
+* [nodes] Add [xod/debug/table-log](https://xod.io/libs/xod/debug/table-log) node to collect tabular data (#2098, #2108)
+* [infra] Pin node versions in CircleCI runners (#2093)
```

Comment:
I do not think `infra` entries should be in the changelog if they don't affect the end user.

Reply:
👌